mscore/workspace.cpp

```diff
@@ -792,6 +794,8 @@ void Workspace::readGlobalMenuBar()
                         if (e.name() == "MenuBar") {
                               QMenuBar* mb = mscore->menuBar();
                               mb->clear();
+                              for (QObject* o : mb->children())
+                                    delete o;
```

Comment:
Would it work to use qDeleteAll() instead?

Reply:
No, it seems to be important also to filter only children of QMenu type, otherwise I get a crash on startup. See [this commit](https://github.com/musescore/MuseScore/pull/5194/commits/6a5a7cbcab30c94d2fcb0801670ba4ea694eb9c7) for the modified version of this patch.